lib/widgets/action_sheet.dart

```diff
@@ -402,3 +410,27 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MarkAsUnreadButton extends MessageActionSheetMenuItemButton {
```

Comment:
nit: order classes to match how they appear in the menu, as listed at the top of the file — that makes it a bit easier to navigate around the file